frontend/package.json

```diff
@@ -8,8 +8,10 @@
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview --port 5173",
-    "cypress:test": "cypress run",
-    "test": "npm run cypress:test"
+    "cy:test": "cypress run",
+    "cy:open": "cypress open",
+    "coverage": "vitest run --coverage",
+    "test": "vitest run"
```

Comment:
Make sure that `npm run test` runs all tests, component and e2e

I think you can add `vitest run && npm run cy:test` for this.

Reply:
[ee1992f](https://github.com/SELab-2/UGent-3/pull/290/commits/ee1992fbd9fab84640246cd8651974235b2a52fa)